### DISCUSSION TITLE
Navigation Links have Disapeared for non Admin users

### DISCUSSION THREAD

**Discussion by swing18** [Budibase/budibase#7209] [Help]
I've noticed in my app that for all non admin users, the links to other screens in my navigation panel have disappeared.  However, on my end as admin I can still access them. 

These users have permission to access all the links that should be in the navigation bar (none are set to admin only), show navigation is selected for each screen in screen settings, and as a test I've been able to change and publish all other parameters in my navigation panel with non admin users seeing these changes. 

Using the app preview and previewing the app as a power or basic user also does not reflect the issue I'm describing. (the links show up fine). 

Has anyone else had this issue and knows how to fix it?

**Comment by swing18** [comment]
changing these links to the "minnimum role" option (basic dosent have the same effect) lets the non admin users view them. but this obviously isn't a viable solution as my application has multiple use tiers with distinct prmissions.

**Reply by swing18** [reply]
I believe this is stemming from an issue with the auto created users table in the budibase db. The following are bugs I've found that trace back to this table. 
- users role's from this table aren't translating into the screens they should be permitted to see.
- i can no longer use the users table as a datasource with the autogenerated screens 
- i've added a column in my Users table called "Project" which identifies the project each user has been assigned to. The binding "{{ Current User.Project }}"  returns nothing for non admin users.

**Comment by melohagan** [comment]
Hi @swing18 

Do you mean that the links appear in the preview for power and basic users, but do not appear in your published app? 

It might help to see some screenshots, or if you can share your app export with: support@budibase.com
